charts/patch/templates/patch.json

```diff
@@ -67,7 +67,7 @@
       {{- include "partials.proxy-init" . | fromYaml | toPrettyJson | nindent 6 }}
   },
   {{- end }}
-  {{- with .Values.debugContainer }}
+  {{- if .Values.debugContainer }}
```

Comment:
Why this change?

Reply:
Oh I see, the partial now needs access to all the values in order to use `default` :+1: